skiko/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.13-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.14.3-all.zip
```

Comment:
I'd keep it 8.13-bin across the projects for now (in case if there is no explicit reason)

Reply:
@MatkovIvan sure, let me double check both this and kotlin version after it was updated in seiko